src/models/backup.js

```diff
@@ -363,14 +363,13 @@ export default {
       }
     },
     updateBackgroundBackups(state, action) {
-      let volumeName = getBackupVolumeName(state.search)
-      if (volumeName && action.payload && action.payload.data) {
-        const backupData = action.payload.data.filter((item) => {
-          return item.volumeName === volumeName
-        })
+      const volumeId = state.search?.keyword
+      const targetBackupVolume = state.backupVolumes.find((item) => item.name === volumeId) || {}
+      if (targetBackupVolume?.volumeName && action.payload?.data) {
+        const backupData = action.payload.data.filter((item) => item.volumeName === targetBackupVolume?.volumeName)
         return {
           ...state,
-          backupData: backupData || [],
+          backupData,
         }
       }
       return {
```

Comment:
Found a issue in backing image page, when accessing `/backingImage` directly, the `Back up` button is disabled.
ref: https://github.com/longhorn/longhorn-ui/pull/755/files#diff-bbfb23ee1c5b3de51151593ac9fe60b487342f1265e1195ac59497efe9f9ed9cR188

Reply:
Fixed in [d68cc66](https://github.com/longhorn/longhorn-ui/pull/830/commits/d68cc666bcf27feaf2bc3d0e7383410a8fc5de14)